src/algorithms/postprocessing/eqOdds.jl

```diff
@@ -17,6 +17,12 @@ function EqOddsWrapper(classifier::MLJBase.Model; grp::Symbol=:class)
 	return EqOddsWrapper(grp, classifier)
 end
 
+function MLJBase.clean!(model::EqOddsWrapper)
+    warning = ""
+    target_scitype(model) <: AbstractVector{<:Finite} || (warning = "Only Binary Classifiers are supported")
+    return warning
+end
+
 # Corresponds to eq_odds_optimal_mix_rates function, mix_rates are returned as fitresult
```

Comment:
This should be `Finite{2}` not `Finite`

Reply:
I had tried keeping it as Finite{2} and not Finite. But models which work perfectly with the wrappers and fairness data return false when I use Finite{2}. But work perfectly with Finite.
For eg.
`model = @pipeline ContinuousEncoder @load(RandomForestClassifier, pkg=DecisionTree)` works perfectly. But it gives false for `target_scitype(model) <: AbstractVector{<:Finite{2}}` but true for `target_scitype(model) <: AbstractVector{<:Finite}`

Reply:
The same thing happens in the case of other wrappers below as well